main.cpp

```diff
@@ -54,6 +59,7 @@ struct CLICallbacks
 		callbacks[cli] = func;
 	}
 	unordered_map<string, function<void(CLIParser &)>> callbacks;
+	uint64_t pad0;
```

Comment:
Removed data padding in this commit: https://github.com/KhronosGroup/SPIRV-Cross/pull/117/commits/5e5f93c2e7e1c0a521b97c2226244234839a9027